front/src/applications/operationalStudies/views/Study.tsx

```diff
@@ -65,12 +80,27 @@ export default function Study() {
   const [scenariosList, setScenariosList] = useState<ScenarioResult[]>([]);
   const [filter, setFilter] = useState('');
   const [filterChips, setFilterChips] = useState('');
-  const [sortOption, setSortOption] = useState('LastModifiedDesc');
+  const [sortOption, setSortOption] = useState<SortOptions>('LastModifiedDesc');
   const [studyStates, setStudyStates] = useState<StateType[]>([]);
   const dispatch = useDispatch();
   const navigate = useNavigate();
-  const projectID = useSelector(getProjectID);
-  const studyID = useSelector(getStudyID);
+  const projectId = useSelector(getProjectID);
+  const studyId = useSelector(getStudyID);
+  const {
+    data: currentProject,
+    isLoading: isProjectLoading,
+    isError: isProjectError,
+    error: projectError,
+  } = osrdEditoastApi.useGetProjectsByProjectIdQuery({ projectId: projectId as number });
+  const [getCurrentStudy] = osrdEditoastApi.useLazyGetProjectsByProjectIdStudiesAndStudyIdQuery();
+  const [postSearch] = osrdEditoastApi.usePostSearchMutation();
+  const [getScenarios] =
+    osrdEditoastApi.useLazyGetProjectsByProjectIdStudiesAndStudyIdScenariosQuery();
+
+  if (isProjectError) {
+    console.error('getProject Error : ', projectError);
+    return <span className="mt-5">An error occured</span>;
+  }
```

Comment:
Same as above : no text in code, fire a notification toast

Reply:
Done